lib/profiler.js

```diff
@@ -97,19 +99,21 @@ function writeProfileFiles(profilePath, options) {
       /* after %~~%profile%~~% before %~~%versions%~~% */ htmlParts[2] +
       versions +
       /* after %~~%versions%~~% */ htmlParts[4];
-    writeFileSync(profilePath + ".html", htmlResult);
+    const htmlPath = profilePath + ".html";
+    writeFileSync(htmlPath, htmlResult);
+    open && opn(htmlPath);
     console.log(
       "⏱️  Profile written to ",
       path.relative(process.cwd(), profilePath + ".html")
     );
   }
 
-  cpuProfiler(profilePath, Object.assign({}, options, { onProfileDone }));
+  cpuProfiler(profilePath, Object.assign({}, opts, { onProfileDone }));
 }
 
-module.exports = (profilePath, options) => {
+module.exports = options => {
```

Comment:
Why do you want to merge this?

Reply:
it is best practice to putting default parameters to be the last of parameters. (`open` defaults to `true`, and options is the last usually). refs: https://eslint.org/docs/rules/default-param-last

I can revert the change if you don't like it. :)

Reply:
I see but why is `profilePath` a default here?